hackathon_site/hardware/serializers.py

```diff
@@ -19,6 +19,7 @@ class Meta:
             "picture",
             "categories",
             "quantity_remaining",
+            "order_items_count"
         )
 
     @staticmethod
```

Comment:
Make sure you remove the `get_quantity_remaining` method, since you've now replaced it